src/Integration.Vsix.UnitTests/CFamily/CFamilyEmbeddedSonarWayRulesTests.cs

```diff
@@ -44,12 +44,14 @@ public class CFamilyEmbeddedSonarWayRulesTests
         // The QP filter has "active/inactive" tabs. The number of rules is shown in the top-right of the screen.
         // 5. Repeat for C++.
 
-        // Rule data for C-Family plugin v6.41 (build 60884)
-        private const int Active_C_Rules = 209;
-        private const int Inactive_C_Rules = 102;
+        // Rule data for C-Family plugin v6.47.0.62356
 
-        private const int Active_CPP_Rules = 435;
-        private const int Inactive_CPP_Rules = 165;
+        // Note: does not include MISRA rules - see #4604
+        private const int Active_C_Rules = 210;
+        private const int Inactive_C_Rules = 123;
```

Comment:
![image](https://github.com/SonarSource/sonarlint-visualstudio/assets/10757559/481f0337-fb99-49b8-93ea-943f3f240205)

The 129 inactive rules for C overall are 123 inactive from the C family plugin + 6 inactive "Common" rules (see next comment).